### ISSUE TITLE
Windows Support

### ISSUE THREAD

**Issue by abhisheksoni27** [abhisheksoni27/codespell#2]
A good starting point: `tasklist` https://technet.microsoft.com/en-us/library/bb491010.aspx

**Comment by 1j01** [comment]
I've created a node module [`process-tree`](https://www.npmjs.com/package/process-tree) that works cross-platform.
It might not have great documentation, but it should be pretty simple to use.

**Comment by abhisheksoni27** [comment]
@1j01 Thank you for that awesome module, Sir.

Would you like to contribute to codespell?

**Comment by 1j01** [comment]
I haven't used the tool, and I'm probably not going to get any benefit from it if it can't [track the active window](https://github.com/abhisheksoni27/codespell/issues/6). I've mainly been using Cloud9 IDE recently, and I generally leave editors open, so it being open isn't gonna be a very useful statistic, unless I change that and get in a habit of closing things.

I've realized `process-tree` would need to be able to return all the processes to be helpful here, not just one that matches a pid, which is something I'd [planned](https://github.com/1j01/process-tree/blob/d99ccfb22492d8170ad76889afc388aefe11347a/index.coffee#L4) but not implemented; and it should be fairly trivial, but the whole module is fairly trivial, and we don't need to list child processes in a tree, so it might make more sense to copy over some of the code into this project, porting it to JS.
(I've started rewriting it in JS.)

**Comment by abhisheksoni27** [comment]
Exactly. We don't need the tree structure. I guess it'd be easier and quicker if we could port the idea behind process-tree to this project, as you said.
